Source/WebInspectorUI/UserInterface/Models/DOMNode.js

```diff
@@ -273,10 +273,8 @@ WI.DOMNode = class DOMNode extends WI.Object
         layoutFlags ||= [];
         console.assert(Array.isArray(layoutFlags), layoutFlags);
         console.assert(layoutFlags.every((layoutFlag) => Object.values(WI.DOMNode.LayoutFlag).includes(layoutFlag)), layoutFlags);
-        console.assert(layoutFlags.filter((layoutFlag) => WI.DOMNode._LayoutContextTypes.includes(layoutFlag)).length <= 1, this._layoutFlags);
-
-        if (Array.shallowEqual(layoutFlags, this._layoutFlags))
-            return;
+        console.assert(layoutFlags.filter((layoutFlag) => WI.DOMNode._LayoutContextTypes.includes(layoutFlag)).length <= 1, layoutFlags);
```

Comment:
yes because this `console.assert` is about checking the incoming `layoutFlags`, not the previous value of `this._layoutFlags`

the next `console.assert` is about making sure that this incoming `layoutFlags` and previous `this._layoutFlags` are not equal, so both are useful to have (tho yes technically we could drop the `this._layoutFlags` since the only way we'd log the value is if they are equal, so no need to log the value twice)